swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, category = "", condition = None):
+        self.category = category
+        if condition is None:
+            condition = 0
+        self.condition = condition 
```

Comment:
Zero is an immutable value, so it can be used as a default:
```suggestion
    def __init__(self, category = "", condition = 0.0):
        self.category = category
        self.condition = condition 
```